lib/web_ui/dev/pipeline.dart

```diff
@@ -201,6 +201,12 @@ class PipelineWatcher {
     io.stdin.echoMode = false;
     io.stdin.lineMode = false;
 
+    // Reset these settings when the felt command is done.
+    cleanupCallbacks.add(() async {
+      io.stdin.echoMode = true;
+      io.stdin.lineMode = true;
+    });
```

Comment:
I like that the restoration is scheduled immediately after the values have been changed!